apps/journeys-admin/src/components/JourneyView/Properties/Properties.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import { MockedProvider } from '@apollo/client/testing'
+import { render, fireEvent, waitFor } from '@testing-library/react'
+import { SnackbarProvider } from 'notistack'
+import { JourneyProvider } from '@core/journeys/ui'
+import { defaultJourney } from '../data'
+import { Properties } from '.'
+
+jest.mock('@mui/material/useMediaQuery', () => ({
+  __esModule: true,
+  default: () => true
+}))
+
+describe('JourneyView/Properties', () => {
+  it('should handle edit journey slug', async () => {
+    const { getAllByRole, getByRole } = render(
+      <SnackbarProvider>
+        <MockedProvider mocks={[]}>
+          <JourneyProvider value={{ journey: defaultJourney, admin: true }}>
+            <Properties />
+          </JourneyProvider>
+        </MockedProvider>
+      </SnackbarProvider>
+    )
+
+    fireEvent.click(getAllByRole('button', { name: 'Edit' })[0])
+    expect(getByRole('dialog')).toBeInTheDocument()
+    fireEvent.click(getByRole('button', { name: 'Cancel' }))
+    await waitFor(() => expect(getByRole('dialog')).toBeInTheDocument())
```

Comment:
Is this what you intended this to check? Maybe this test should just be "should show edit slug dialog"

```suggestion
    await waitFor(() => expect(getByRole('dialog')).not.toBeInTheDocument())
```